src/azurelinux/3.0/crossdeps-builder/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM azurelinuxpreview.azurecr.io/public/azurelinux/base/core:3.0
+
+RUN tdnf install -y \
+        # Common utilities
+        ca-certificates \
+        git \
+        tar \
+        wget \
+        # LLVM build dependencies
+        binutils \
+        clang \
+        cmake \
```

Comment:
We should double check with @jkoritzinsky on the CMake version.

Reply:
This cmake version should be all good (3.28.2)